spec.md

```diff
@@ -549,6 +549,19 @@ Analog<32> ; 32-bit analog type
 Analog     ; analog type with inferred width
 ```
 
+### Enumeration Types
+
+Enumerations are structural disjoint union types.  An enumeration has a number
+of variants, each with a type.  The different variants are specified with tags.
+A variant may optionally omit the data type, in which case it is implicitly
+defined to be `UInt<0>`. The variant types of an enumeration must all be
+passive and cannot contain analog types.
```

Comment:
Nits:

The third sentence uses `data type` when the second sentence uses `type`. Using the latter consistently would be better.

Does passivity exclude probe types or should this be explicitly called out?

Reply:
Fixed + `contain analog or probe types.`